include/prism.h

```diff
@@ -43,6 +43,7 @@
  * The prism version and the serialization format.
  *
  * @returns The prism version as a constant string.
+ * \public \memberof pm_parser
```

Comment:
I would prefer that this not be a member of parser, since it does not accept it as a parameter/doesn't correspond to a parser.